server/scripts/2020TaxSeasonDataScript.php

```diff
@@ -0,0 +1,409 @@
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Southeast Community College data', MY_EXCEPTION);
+		die();
+	}
+}
+
+/*
+ * Helper Methods
+ */ 
+function insertSite($title, $address, $phoneNumber, $doesMultilingual, $doesInternational) {
+	GLOBAL $DB_CONN;
+
+	$query = 'INSERT INTO Site (title, address, phoneNumber, doesMultilingual, doesInternational, createdBy, lastModifiedBy)
+		VALUES (?, ?, ?, ?, ?, 1, 1);';
+	$params = array($title, $address, $phoneNumber, $doesMultilingual, $doesInternational);
+
+	$stmt = $DB_CONN->prepare($query);
+	if (!$stmt->execute($params)) {
+		throw new Exception("Unable to insert site", MY_EXCEPTION);
+	}
+
+	return $DB_CONN->lastInsertId();
+}
+
+function insertShift($startTime, $endTime, $siteId) {
+	GLOBAL $DB_CONN;
+
+	$query = 'INSERT INTO Shift (startTime, endTime, siteId, createdBy, lastModifiedBy)
+		VALUES (?, ?, ?, 1, 1)';
+	$params = array($startTime, $endTime, $siteId);
+
+	$stmt = $DB_CONN->prepare($query);
+	if (!$stmt->execute($params)) {
+		throw new Exception("Unable to insert shift", MY_EXCEPTION);
+	}
+
+	return $DB_CONN->lastInsertId();
+}
+
+function insertAppointmentTime($scheduledTime, $minimumNumberOfAppointments, $maximumNumberOfAppointments, $percentageAppointments, $approximateLengthInMinutes, $siteId) {
+	GLOBAL $DB_CONN;
+
+	$query = 'INSERT INTO AppointmentTime (scheduledTime, minimumNumberOfAppointments, maximumNumberOfAppointments, percentageAppointments, approximateLengthInMinutes, siteId)
+		VALUES (?, ?, ?, ?, ?, ?)';
+	$params = array($scheduledTime, $minimumNumberOfAppointments, $maximumNumberOfAppointments, $percentageAppointments, $approximateLengthInMinutes, $siteId);
+
+	$stmt = $DB_CONN->prepare($query);
+	if (!$stmt->execute($params)) {
+		throw new Exception("Unable to insert appointment time", MY_EXCEPTION);
+	}
+
+	return $DB_CONN->lastInsertId();
+}
+
+function insertSiteRoleLimit($maximumNumber, $roleId, $siteId) {
+	GLOBAL $DB_CONN;
+
+	$query = 'INSERT INTO RoleLimit (maximumNumber, roleId, siteId)
+		VALUES (?, ?, ?)';
+	$params = array($maximumNumber, $roleId, $siteId);
+
+	$stmt = $DB_CONN->prepare($query);
+	if (!$stmt->execute($params)) {
+		throw new Exception("Unable to insert role limit", MY_EXCEPTION);
+	}
+
+	return $DB_CONN->lastInsertId();
+}
+
+function insertShiftRoleLimit($maximumNumber, $roleId, $shiftId, $siteId) {
+	GLOBAL $DB_CONN;
+
+	$query = 'INSERT INTO RoleLimit (maximumNumber, roleId, shiftId, siteId)
+		VALUES (?, ?, ?, ?)';
+	$params = array($maximumNumber, $roleId, $shiftId, $siteId);
+
+	$stmt = $DB_CONN->prepare($query);
+	if (!$stmt->execute($params)) {
+		throw new Exception("Unable to insert role limit", MY_EXCEPTION);
+	}
+
+	return $DB_CONN->lastInsertId();
+}
+
+// Gets dates of the same days of the week as $start, ends on
+// the first occurrence of that day of the week on or after $end
+function getWeeklyDatesFromRange($start, $end, $format = 'Y-m-d') { 

```

Comment:
We might consider renaming the variables to `$startDateInclusive`, `$endDateInclusive`, and `$dateFormat` just to be slightly more clear--especially with the whole inclusive/exclusive thing.

Reply:
Done